km/km_filesys.c

```diff
@@ -75,6 +77,58 @@ static int km_fs_g2h_readlink(const char* name, char* buf, size_t bufsz);
 km_fd_dup_data_t dup_data;
 pthread_mutex_t dup_data_mtx;
 
+/*
+ * Lightweight snap start - wait for connection on the snap_listen_sock before restoring the snapshot.
+ */
+static const_string_t KM_SNAP_LISTEN_PORT = "SNAP_LISTEN_PORT";
```

Comment:
IMO we need to centralize these environment variables in a place where we can provide some in-line documentation.

Reply:
Agree, but I'd say it's a separate PR